images/router/haproxy/conf/haproxy-config.template

```diff
@@ -213,6 +213,9 @@ frontend public
   bind :{{ env "ROUTER_SERVICE_HTTP_PORT" "80" }}{{ if isTrue (env "ROUTER_USE_PROXY_PROTOCOL") }} accept-proxy{{ end }}
     {{- end }}
   mode http
+
+  # https://issues.redhat.com/browse/OCPBUGS-32044
+  option idle-close-on-response
```

Comment:
If you provide a bare URL or bug number with no further information, then the reader has to look up the bug report.  If the reader has to do a lookup, then the reader could just as easily use git-blame to look up the commit message, which should explain the change and have a reference to the bug report.  Could you add more details to the comment, or just remove the comment entirely rely on the commit message to explain the change?
```suggestion
  # Work around an issue with Apache HttpClient 4.5's connection pooling manager,
  # which may attempt to re-use closed idle connections if this option is not enabled.
  option idle-close-on-response
```

Reply:
I didn't give this diff enough of care yet. I just dropped the option to be able to test it. If this change will be the way to go I think I'll add some text as you proposed.

Btw the issue is not only with the Apache HTTP Client, initially the option was implemented for AWS because the ALBs kept sending requests to the closed (before idle ) connections (https://github.com/haproxy/haproxy/issues/1506).

Reply:
The comment is updated. PTAL.